### PR TITLE
Feature: add `makeMemoizedValidator`

### PR DESCRIPTION
**Pull request by jvhoven**
- [ ] Release 1.0.1 
- [ ] Update devDependency in `@42.nl/ui`
- [ ] Update dependency in seeder
- [ ] Test in seeder

### CODE REVIEW

**Comment by MrHus** [comment]
@jvhoven ready for review

**Review by jvhoven — COMMENTED** [review]
Reviewed, however I can't approve it, as it is my own pull request.

**Review by jvhoven — COMMENTED** [review]
Second review, can't approve as I'm the author of the PR.

**Review by MrHus — APPROVED** [review]

**Pull request merged**
